src/transformStateWithClones.js

```diff
@@ -6,8 +6,34 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
   // write code here
+  let stateClone = { ...state };
+  const stateCopy = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        stateClone = {
+          ...stateClone, ...action.extraData,
+        };
+
```

Comment:
No need for an empty lines before break
You can remove it next to break to make cases visually separated